src/cskvp.rs

```diff
@@ -160,6 +160,10 @@ impl<'a> Cskvp<'a> {
         self.double.remove(key)
     }
 
+    pub fn take_all_double(&mut self) -> HashMap<Cow<'a, str>, WithRange<Cow<'a, str>>> {
```

Comment:
In `Drop` we warn for all unused attributes of a `Cskvp`. Using this method makes that more or less pointless.